docs/sources/configuration/examples.md

```diff
@@ -333,3 +333,55 @@ storage_config:
       kms_key_id: 0987dcba-09fe-87dc-65ba-ab0987654321
 ```
 
+
+## 11-IBM-Cloud-Object-Storage-Config-With-HMAC.yaml
+
+```yaml
+
+# This partial configuration uses IBM Cloud Object Storage (COS) for chunk storage. HMAC will be used for authenticating with COS.
+
+schema_config:
+  configs:
+    - from: 2020-10-01
+      index:
+        period: 24h
+        prefix: loki_index_
+      object_store: "cos"
+      schema: v11
+      store: "boltdb-shipper"
+
+storage_config:
+  cos:
+    bucket: <bucket>
```

Comment:
We need to fix the bucket field name.`bucket` vs `bucketnames`.

Reply:
We discussed and agreed to be consistent with s3 and keep `bucketnames`.